pkg/utils/helm/helm_helper.go

```diff
@@ -351,3 +357,33 @@ func calculateCacheTimeFromIndex(length int) time.Duration {
 	}
 	return cacheTime
 }
+
+// nolint
+func fetchChartValuesFromOciRepo(repoURL string, chartName string, version string, opts *common.HTTPOption) (map[string]interface{}, error) {
+	d := downloader.ChartDownloader{
+		Verify:  downloader.VerifyNever,
+		Getters: getter.All(cli.New()),
+		Options: []getter.Option{
+			getter.WithBasicAuth(opts.Username, opts.Password),
+			getter.WithInsecureSkipVerifyTLS(opts.InsecureSkipTLS),
+			getter.WithTLSClientConfig(opts.CertFile, opts.KeyFile, opts.CaFile),
+		},
+	}
+	var err error
+	dest, err := os.MkdirTemp("", "helm-")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to fetch values file")
+	}
+	defer os.RemoveAll(dest)
+
+	chartRef := fmt.Sprintf("%s/%s", repoURL, chartName)
+	saved, _, err := d.DownloadTo(chartRef, version, dest)
+	if err != nil {
+		return nil, err
+	}
+	c, err := loader.Load(saved)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to fetch values file")
+	}
+	return c.Values, nil
```

Comment:
Can we parse the description of any values?

Reply:
The values struct doesn't have this value.

Reply:
Let's return the content of the `value.yaml` file, It could get from the field `c.Raw`.
Then we could like the `helm show values` command, show the content of the value.

Reply:
Done